### PR TITLE
Refactored env config to use shared library

### PR DESCRIPTION
**Pull request by karelklima**

### CODE REVIEW

File: src/app/variables.ts [hunk 1]
```diff
@@ -1,75 +1,48 @@
-import YAML from 'yaml'
+import { Env } from '@opendata-mvcr/assembly-line-shared'
```
Comment (psiotwo) [inline src/app/variables.ts:1]: we agreed on the double-quotes, but this will be handled by another PR I expect.

**Review by psiotwo — APPROVED** [review]
lgtm

File: src/app/variables.ts (hunk 1 above)
Comment (karelklima) [reply src/app/variables.ts:1]: Yes!
